lib/dynamo-backup.js

```diff
@@ -65,7 +74,9 @@ DynamoBackup.prototype.backupTable = function (tableName, backupPath, callback)
         },
         function(err) {
             stream.end();
-            callback(err);
+            if(err) {
```

Comment:
This isn't quite right.  If `_copyTable` could return an error before we've even started streaming.  I think we should emit an error here, if an error was returned.